go/service/delegate_ui.go

```diff
@@ -47,3 +47,14 @@ func (d *DelegateUICtlHandler) RegisterUpdateUI(_ context.Context) error {
 	d.G().UIRouter.SetUI(d.id, libkb.UpdateUIKind)
 	return nil
 }
+
+func (d *DelegateUICtlHandler) RegisterRekeyUI(_ context.Context) error {
+	d.G().UIRouter.SetUI(d.id, libkb.RekeyUIKind)
+
+	// Let Gregor related code know that a RekeyUI client
```

Comment:
But isn't the whole point of the RekeyUIHandler that once electron connects it will sync and show any rekey needed messages?  If we do this at service startup, there could be rekey messages in the service log and nothing when electron connects.